cg/models/cg_config.py

```diff
@@ -170,6 +170,14 @@ class RnafusionConfig(CommonAppConfig):
 class TaxprofilerConfig(CommonAppConfig):
```

Comment:
remember that if you update this you need to update the config in servers

Reply:
I have this [PR](https://github.com/Clinical-Genomics/servers/blob/d401fbcdc93dafc81d64791185a0a7e61cb080e8/config/hasta.scilifelab.se/cg-stage.yaml) open that I will merge once this one is approved.
 
I have added `pipeline_path`, `revision`, `hostremoval_reference`, `databases`.